src/contracts/DependencyInjection/Configuration/SiteAccessAware/IgnoredConfigParser.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Contracts\Test\Core\DependencyInjection\Configuration\SiteAccessAware;
+
+use Ibexa\Bundle\Core\DependencyInjection\Configuration\AbstractParser;
+use Ibexa\Bundle\Core\DependencyInjection\Configuration\SiteAccessAware\ContextualizerInterface;
+use Symfony\Component\Config\Definition\Builder\NodeBuilder;
+
+/**
+ * SA configuration parser which allows ignoring given keys in tests.
+ */
+final class IgnoredConfigParser extends AbstractParser
+{
+    /** @var string[] */
+    private array $keys;
+
+    /**
+     * @param string[] $keys
+     */
+    public function __construct(array $keys)
```

Comment:
How about
```suggestion
    public function __construct(string ...$keys)
```
which would change the usage to
```php
final class IbexaTestKernel extends BaseIbexaTestKernel
{
    # ...

    public function registerContainerConfiguration(LoaderInterface $loader): void
    {
        $loader->load(static function (ContainerBuilder $container): void {
            /** @var \Ibexa\Bundle\Core\DependencyInjection\IbexaCoreExtension $ibexa */
            $ibexa = $container->getExtension('ibexa');
            $ibexa->addConfigParser(new IgnoredConfigParser(
                'admin_ui_forms',
                'content_create_view',
                'content_translate_view',
                'content_edit_view',
                'limitation_value_templates',
                'universal_discovery_widget_module',
                'page_builder_forms',
                'design',
                'user_registration',
            ));
        });
    }

    #...
}
```